pkg/apiclient/apiclient.go

```diff
@@ -711,9 +711,6 @@ func (c *client) WatchApplicationWithRetry(ctx context.Context, appName string,
 				if isCanceledContextErr(err) {
 					cancelled = true
 				} else {
-					if err != io.EOF {
```

Comment:
Watch is often closed due to load balancers forcefully close connection after some time. The warning message only confuses users and should not be printed.